src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -235,7 +235,7 @@ The Machine Learning Open Studio provides a fast, easy and practical way to exec
 
 To test these workflows, you need to add the *machine-Learning-workflows Bucket* as main catalog in the ProActive Studio.
 
-A. Open http://try.activeeon.com/studio[ProActive Workflow Studio^] home page.
+A. Open link:/studio[ProActive Workflow Studio^] home page.
```

Comment:
this is a tricky part because it will break the link in the online documentation (which should still be valid). I don't know well how we can handle this kind of issue.